src/nav/alerts-applied-intelligence.yml

```diff
@@ -79,7 +79,7 @@ pages:
             path: /docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/muting-rules-suppress-notifications
   - title: Incident response
     pages:
-      - title: Issue and incident management and response
+      - title: Issues, incident management,and response
```

Comment:
As far as the title goes, we don't really mention incidents in the doc outside of the specific UI elements, maybe the title should be reworked or we should go into incidents a little more? What I mean is if I come in to the doc looking for incidents specifically (for some reason), it doesn't appear in the right nav at all and if I'm not familiar with this section of the product, I feel like the lack of "incident" callouts could cause irritation for the reader.